packages/fluentui/react-bindings/src/hooks/useFrameFocusDispatch.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+export const useFrameFocusDispatch = (
+  enableFrameFocusDispatch: boolean,
+  $document: Document,
```

Comment:
```suggestion
  targetDocument: Document,
```

_nit_ I suggest to use `targetDocument` to be consistent with other parts of code.